.github/workflows/main.yml

```diff
@@ -17,63 +17,63 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v3
-
-    - name: Set up JDK 8
-      uses: actions/setup-java@v3
-      with:
-        java-version: '8'
-        distribution: 'temurin'
-
-    - name: Build with Maven
-      run: mvn -B package -Dmaven.test.skip=true --file pom.xml
-
-    - name: Run tests
-      env:
-        ACCESS_KEY: ${{ secrets.DEV_CA_PUBLIC_USE_INTEGRATION_TEST_ACCESS_KEY }}
-        SERVICE_PRINCIPAL_KEY: ${{ secrets.DEV_CA_PUBLIC_USE_TESTOAUTHSERVICEPRINCIPAL_SERVICE_PRINCIPAL_KEY }}
-        REPOSITORY_ID:  ${{ secrets.DEV_CA_PUBLIC_USE_REPOSITORY_ID_1 }}
-        TEST_HEADER: ${{ secrets.TEST_HEADER }}
-      run: mvn test
-
-    - name: Publish Test Results
-      uses: EnricoMi/publish-unit-test-result-action@v1
-      if: always()
-      with:
-        files: "target/surefire-reports/*.xml"
+      - uses: actions/checkout@v3
```

Comment:
It seems that this is just indentation change. Do we want to change the indentation?